src/macosMain/kotlin/co/touchlab/xcode/cli/main.kt

```diff
@@ -6,23 +6,36 @@ import co.touchlab.xcode.cli.command.Info
 import co.touchlab.xcode.cli.command.Install
 import co.touchlab.xcode.cli.command.Repair
 import co.touchlab.xcode.cli.command.Uninstall
+import co.touchlab.xcode.cli.util.CrashHelper
 import kotlinx.cli.ArgParser
 
 fun main(args: Array<String>) {
-    // If no arguments were given, we want to show help.
-    val adjustedArgs = if (args.isEmpty()) {
-        arrayOf("-h")
-    } else {
-        args
-    }
-    Logger.setLogWriters(platformLogWriter())
-    val parser = ArgParser("xcode-kotlin")
-    parser.subcommands(
-        Install(),
-        Uninstall(),
-        Repair(),
-        Info(),
-    )
+    val crashHelper = CrashHelper()
```

Comment:
Everything in this app is a singleton (`object`), it might make sense to do the same for the `CrashHelper`, what do you think?

Reply:
I think it's reasonable, but you need a `pp` on this ;) I don't care much where it lands. I'm starting to see classes and objects as simple function holders now that we're all a little post-OOP in our thinking.

Reply:
If you meant I need `pp` on whether it's all `class` or `object` then I agree. If you mean whether this one should be `class` or `object` in the context where everything else is an `object` then I disagree (or more than likely I misunderstood `pp`s). Happy to discuss off this PR.

Reply:
Adding a ticket about this to discuss next time, related to project-specific architecture decisions. https://github.com/touchlab/Touchlab/issues/191. If the same code would get different reviews in a different project, it's at least project-specific, if not `pp`.

Reply:
Yeah it's definitely project specific, no argument there.